app/databasemanager.py

```diff
@@ -1,32 +1,148 @@
+"""Module containing Classes and functions relating to managing
+the database."""
+
 import sqlite3
 from sqlite3 import Connection, Error
+from dataclasses import dataclass
+from models.rowdata import RowData
+import pandas as pd
+
 
+DEFAULT_DATABASE = 'sqlitedatabase.db'
 
-DEFAULT_DATABASE = "sqlitedatabase.db"
+
+@dataclass
+class Table:
+    """Class to hold column and table names of the table in the database"""
+    NAME = 'data'
+    COLUMN_ENTITY = 'entity'
+    COLUMN_YEAR = 'year'
+    COLUMN_PREVALENCE = 'prevalence'
 
 
 class DatabaseManager:
     """Class to manage data stored in an SQLite database"""
-    def __init__(self, db=DEFAULT_DATABASE) -> None:
-        self.db = db
-        self.connection = self.create_connection()
 
+    def __init__(self, database=DEFAULT_DATABASE) -> None:
+        self.database = database
+        self.connection = self.create_connection()
 
     def create_connection(self) -> Connection:
```

Comment:
change return type to `Optional[Connection]` or `Union[Connection, None]` to make it explicit that null could be returned.